pkg/controller/migstorage/migstorage_controller.go

```diff
@@ -82,6 +82,8 @@ type ReconcileMigStorage struct {
 // +kubebuilder:rbac:groups=migration.openshift.io,resources=migstorages,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=migration.openshift.io,resources=migstorages/status,verbs=get;update;patch
 func (r *ReconcileMigStorage) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+	log.Info("[mStorage] RECONCILE() ", "request", request)
```

Comment:
Ah interesting, so does this show the request ns/name as a key/value in the log message?